eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -64,6 +64,37 @@ jobs:
         parameters:
           ServiceDirectory: ${{parameters.ServiceDirectory}}
 
+      - task: DownloadPipelineArtifact@2
+        inputs:
+          buildType: "current"
+          artifactName: packages
+          itemPattern: "*.tgz"
+          targetPath: "$(Agent.TempDirectory)/packagesCurrent"
+        displayName: "Download Current Build artifacts"
+
+      - task: DownloadPipelineArtifact@2
```

Comment:
Does this require any special permissions?

Reply:
No it required the project Guid and system.definitionId, doesn't require any permissions